registration/registration-services/src/main/java/io/mosip/registration/mdm/service/impl/MosipDeviceSpecificationFactory.java

```diff
@@ -310,7 +310,7 @@ public String getLatestSpecVersion(String[] specVersion) {
 			}
 
 			if (getMdsProvider(deviceSpecificationProviders, latestSpecVersion) == null) {
-				List<String> specVersions = Arrays.asList(specVersion);
+				List<String> specVersions = new ArrayList<>(Arrays.asList(specVersion));
```

Comment:
Is this required?

Reply:
I have added this to resolve sonar bug. Since Arrays.asList() returns an unmodifiable list, wrapping it here.